file-formats/uist/type/zif_aff_uist_v1.intf.abap

```diff
@@ -0,0 +1,67 @@
+INTERFACE zif_aff_uist_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">SAP Fiori Page Template</p>
+    "! SAP Fiori Page Template
+    BEGIN OF ty_page,
+      "! <p class="shorttext">ID</p>
+      "! ID
+      page_template_id          TYPE c LENGTH 35,
+
+      "! <p class="shorttext">Title</p>
+      "! Title
+      page_template_title       TYPE zif_aff_types_v1=>ty_description_100,
+
+      "! <p class="shorttext">Description</p>
+      "! Description
+      page_template_description TYPE zif_aff_types_v1=>ty_description_100,
+    END OF ty_page,
+
+    ty_pages TYPE TABLE OF ty_page WITH KEY page_template_id.
+
+  TYPES:
+    "! <p class="shorttext">Fiori Launchpad Space Template</p>
+    "! Fiori Launchpad Space Template
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version               TYPE zif_aff_types_v1=>ty_format_version,
+
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                       TYPE zif_aff_types_v1=>ty_header_100,
+
+      "! <p class="shorttext">ID</p>
+      "! ID
+      "! $required
+      space_template_id            TYPE c LENGTH 35,
+
+      "! <p class="shorttext">Title</p>
+      "! Title
+      "! $required
+      space_template_title         TYPE zif_aff_types_v1=>ty_description_100,
+
+      "! <p class="shorttext">Description </p>
+      "! Description
+      "! $required
+      space_template_description   TYPE zif_aff_types_v1=>ty_description_100,
+
+      "! <p class="shorttext">Sort Priority</p>
+      "! Sort priority
+      "! $minimum: 0.001
+      "! $maximum: 999.999
+      "! $multipleOf: 0.001
```

Comment:
Would you remove the annotation `$multipleOf`, please? We face issues with the validation and will update our generator accordingly
```suggestion
      "! $multipleOf: 0.001
```